templates/pages/article_page.html

```diff
@@ -58,10 +58,10 @@ <h1 class="font-serif4 font-bold text-4xl lg:text-7xl">
             />
         </picture>
 
-        
+
 
     </div>
-    {% if value.caption %}
+    {% if page.image.0.value.caption %}
         <p class="text-sm leading-6 py-5">{{ page.image.0.value.caption }}</p>
```

Comment:
Note: I had an earlier commit that replaces this with `figure` and `figcaption`, but I removed it because we might want to do it in #41 instead. I don't know why we write the template for the image block directly on the page template instead of the image block's template.

Maybe because the image block is meant to be more general-purpose rather than a big main image? Or maybe because the code originally used a foreign key and was replaced with a StreamField that only contains the `ImageBlock` 🤔

Reply:
But what if people want to add an `ImageBlock` to their StreamField? Would we encourage them to add images in their article body with `RichTextBlock` instead? We have to think about how editors are adding images to their content further down the line and I feel like it's slightly more intuitive to select an image block from the StreamField menu than to add it via another method.

Reply:
it just occurred to me that the default `ImageBlock` could be called into StreamField. I think the argument for having a separate one though would be more precise control of the styling and how the image is displayed.